src/index.html

```diff
@@ -11,6 +11,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      class="search-form search-form--big"
+      data-qa="big"
+    >
+      <label class="search-form__label">
+        <span class="search-form__image search-form__image--big"></span>
+        <input
+          type="text"
+          class="search-form__input search-form__input--big"
```

Comment:
```suggestion
          class="search-form__input"
```
you don't need --big modifier.
set all styles in .search-form__input selector and change necessary for --small modifier.
same for other elements